client/src/utils/interfaces.ts

```diff
@@ -18,6 +18,7 @@ interface Event {
   latitude?: number;
   longitude?: number;
   organizerId?: number;
+  distance?: number;
```

Comment:
it's confusing to put computed value like `distance` in the Event interface, which should contains fields intrinsic to the Event.

Reply:
probably you would like to separate/clarify the interfaces, for example, one `EventModel` interface that mirrors the DB schema, and one `DisplayEvent` for the presentational layer.